media/js/mozorg/contact-spaces.js

```diff
@@ -416,6 +418,7 @@
                 }
             } else if (state === 'communities') {
                 initContentId = $('#nav-communities li.current').data('id');
+                mozMap.setMapState();
```

Comment:
I think here you might want to use something like:

    var region = initContentId ? initContentId : 'communities';
mozMap.showCommunityRegion